modules/manually-create-identity-access-management.adoc

```diff
@@ -291,6 +279,20 @@ $ rm 0000_30_capi-operator_00_credentials-request.yaml
 ----
 endif::cco-manual-mode[]
 
+. Copy the generated credential files to the target manifests directory:
++
+[source,terminal]
+----
+$ cp ./<path_to_ccoctl_output_dir>/manifests/*credentials.yaml ./<path_to_installation>dir>/manifests/
+----
++
+where:
+
+ifdef::alibaba-default,alibaba-custom[]
+`<path_to_ccoctl_output_dir>`:: Specifies the directory created by the `ccoctl alibabacloud create-ram-users` command.
+endif::alibaba-default,alibaba-custom[]
+`<path_to_installation_dir>`:: Specifies the directory in which the installation program creates files.
+
 . Create YAML files for secrets in the `openshift-install` manifests directory that you generated previously. The secrets must be stored using the namespace and secret name defined in the `spec.secretRef` for each `CredentialsRequest` object. The format for the secret data varies for each cloud provider.
```

Comment:
We do not need this step if user uses ccoctl to create secret manifests. If the user does not use ccoctl then they need to create these files manually as described in this step. 

IF `secret manifests were created by ccoctl then copy the generated credential files to the target manifests directory.`
ELSE `Create YAML files for secrets in the `openshift-install` manifests directory that you generated previously. The secrets must be stored using the namespace and secret name defined in the `spec.secretRef` for each `CredentialsRequest` object. The format for the secret data varies for each cloud provider.`